Foundation/NSNotificationQueue.swift

```diff
@@ -31,6 +33,16 @@ public class NSNotificationQueue : NSObject {
     internal let notificationCenter: NSNotificationCenter
     internal var asapList = NSNotificationList()
     internal var idleList = NSNotificationList()
+    internal lazy var idleRunloopObserver: CFRunLoopObserverRef = {
+        return CFRunLoopObserverCreateWithHandler(kCFAllocatorDefault, CUnsignedLong(1) << 5, true, 0) {[weak self] observer, activity in
+            self!.notifyQueues(.PostWhenIdle)
+        }
+    }()
+    internal lazy var asapRunloopObserver: CFRunLoopObserverRef = {
+        return CFRunLoopObserverCreateWithHandler(kCFAllocatorDefault, CUnsignedLong(1) << 1 | CUnsignedLong(1) << 7, true, 0) {[weak self] observer, activity in
```

Comment:
It used to be like that. For some reason I got the next compile time error on Linux:

```
Foundation/NSNotificationQueue.swift:37:72: error: type 'CFRunLoopActivity' (aka 'UInt') has no member 'BeforeWaiting'
        return CFRunLoopObserverCreateWithHandler(kCFAllocatorDefault, CFRunLoopActivity.BeforeWaiting.rawValue, true, 0) {[weak self] observer, activity in
```